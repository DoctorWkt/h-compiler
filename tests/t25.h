void main()
{
  int o_ll=000101;
  int m=000000;
  int l=000007;

  printn(o_ll);
  printn(m);
  printn(l);
  if ((o_ll - m) / 2 > l)
        m = m + (o_ll - m) / 2;
  else
        m = m + l;
  printn(m);
}
