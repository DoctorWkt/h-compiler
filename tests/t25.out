000101
000000
000007
000040
